freeipa/src/test/java/com/sequenceiq/freeipa/converter/instance/VmTypeToVmTypeResponseConverterTest.java

```diff
@@ -0,0 +1,65 @@
+package com.sequenceiq.freeipa.converter.instance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
+import static org.junit.Assert.assertEquals;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.cloudbreak.cloud.model.VmType;
+import com.sequenceiq.cloudbreak.cloud.model.VmTypeMeta;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.create.VmTypeMetaJson;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.create.VmTypeResponse;
+import com.sequenceiq.freeipa.api.v1.freeipa.stack.model.create.VolumeParameterConfigResponse;
+
+public class VmTypeToVmTypeResponseConverterTest {
+
+    private VmTypeToVmTypeResponseConverter underTest = new VmTypeToVmTypeResponseConverter();
```

Comment:
Does not make a big difference as converters are expected to be stateless, but either instantiate this in a `@BeforeEach setUp()` method, or make the field `final`.